android/build.gradle

```diff
@@ -65,7 +65,7 @@ android {
 }
 
 dependencies {
-    api('com.onegini.mobile.sdk.android:onegini-sdk:12.0.0@aar') {
+    api('com.onegini.mobile.sdk.android:onegini-sdk:12.1.1@aar') {

```

Comment:
in 12.1.0 we changed the type of the error returned when registation is aborted by developer https://developer.onewelcome.com/android/sdk/v12-X#improvements. Did you check if any changes in the code are needed? (I guess nothing has to be changed) We should also mention that in the release notes for new flutter version

Reply:
No additional code changes required. The new error message "The registration action was canceled by the user" is passed back through Flutter plugin as well.